packages/DataGrid/src/DataGrid.js

```diff
@@ -0,0 +1,628 @@
+    return ColumnDefinitions.length;
+  }, [ColumnDefinitions]);
+
+  const columnHeadersA11yText = React.useMemo(() => {
+    return ColumnDefinitions.map(ColumnDefinition => {
+      const { header, headerA11yText } = ColumnDefinition.props;
+      return typeof header === "function" ? headerA11yText && headerA11yText() : header;
+    });
+  }, [ColumnDefinitions]);
+
+  const calculatedTableHeight = React.useMemo(() => {
+    return rowHeight * rowCount > height ? height : rowHeight * rowCount + scrollBarWidth;
+  }, [height, rowCount, rowHeight, scrollBarWidth]);
+
+  const calculatedTableWidth = React.useCallback(() => {
+    let width = 0;
+    ColumnDefinitions.forEach(ColumnDefinition => {
+      if (!ColumnDefinition.props.isSticky) {
+        width += Number.parseInt(ColumnDefinition.props.width, 10);
+      }
+    });
+
+    return width + scrollBarWidth;
+  }, [ColumnDefinitions, scrollBarWidth]);
+
+  const gridWidth = width === null ? calculatedTableWidth() : width;
+
+  const stickyColumnsIndexes = React.useMemo(
+    () =>
+      ColumnDefinitions.map((ColumnDefinition, index) => {
+        return ColumnDefinition.props.isSticky ? index : null;
+      }).filter(chunk => chunk !== null),
+    [ColumnDefinitions]
+  );
+
+  const stickyGridWidth = React.useMemo(() => {
+    let width = 0;
+    if (stickyColumnsIndexes.length) {
+      stickyColumnsIndexes.forEach(index => {
+        width += ColumnDefinitions[index].props.width;
+      });
+    }
+    return width;
+  }, [ColumnDefinitions, stickyColumnsIndexes]);
+
+  const highlightRow = React.useCallback(({ rowIndex = null }) => {
+    if (rowIndex !== null) {
+      if (refPrevActiveRow.current !== null) {
+        const rowIndex = refPrevActiveRow.current;
+        const key = `0${rowIndex}`;
+        if (refsCell.current.keys[key]) refsCell.current.keys[key].deemphasizeOnrow(rowIndex);
+      }
+
+      refActiveRow.current = rowIndex;
+      refPrevActiveRow.current = rowIndex;
+
+      // first column nth row only
+      const key = `0${rowIndex}`;
+      if (refsCell.current.keys[key]) refsCell.current.keys[key].highlightOnRow(rowIndex);
+    }
+  }, []);
+
+  const deemphasizeRow = React.useCallback(() => {
+    if (refPrevActiveRow.current) {
+      const rowIndex = refPrevActiveRow.current;
+
+      // first column nth row only
+      const key = `0${rowIndex}`;
+      if (refsCell.current.keys[key]) refsCell.current.keys[key].deemphasizeOnrow(rowIndex);
+    }
+
+    refActiveRow.current = null;
+    refPrevActiveRow.current = null;
+  }, []);
+
+  const onChangeActiveCell = React.useCallback(({ columnIndex, rowIndex }) => {
+    const key = `${columnIndex}${rowIndex}`;
+
+    if (refPrevActiveCell.current && refPrevActiveCell.current in refsCell.current.keys) {
+      const prevCell = refsCell.current.keys[refPrevActiveCell.current];
+      if (prevCell) {
+        refsCell.current.keys[refPrevActiveCell.current].isActiveCell(false);
+      }
```

Comment:
After declaring prevCell, I think you meant to use it:

    if (prevCell) {
      prevCell.isActivecell(false);
    }